common/assets/scss/application.scss

```diff
@@ -27,6 +27,7 @@ $govuk-font-family: "Inter", system-ui, sans-serif;
 @import "govuk-frontend/govuk/components/skip-link/skip-link.scss";
 @import "govuk-frontend/govuk/components/summary-list/summary-list.scss";
 @import "govuk-frontend/govuk/components/table/table.scss";
+@import "govuk-frontend/govuk/components/tabs/tabs.scss";
```

Comment:
We can remove the need for these at this stage. This change should be MVP of an allocation.

Reply:
This can be removed for now. I noticed bringing in lots of the GOV.UK Design System is bloating the css file so we should do it sparingly.